package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
     "esn-frontend-mailto-handler": "github:openpaas-suite/esn-frontend-mailto-handler#main",
+    "esn-frontend-videoconference-calendar": "github:piavgh/esn-frontend-videoconference-calendar#bootstrap",
```

Comment:
shouldn't we use https://github.com/OpenPaaS-Suite/esn-frontend-videoconference-calendar instead ( when it's populated with code )?

Reply:
yes, but the other PR was not merged yet :D I noted it and will update this PR when that PR is merged

Reply:
updated it